.github/workflows/dco.yml

```diff
@@ -1,5 +1,7 @@
 name: dco
 on:
+  push:
+    branches: [ main ]
```

Comment:
Include releases as well in case we create release branches?

Reply:
done